### PR TITLE
fix edm_type

### PR DESCRIPTION
**Pull request by jacobthill**
## Why was this change made?

Fix a few errors in LOC el-Taher collection
- remove trailing backslash in title field
- one photograph had a cho_edm_type value of 'Text' instead of 'Image'.

## How was this change tested?

Local transform viewed in the dev site.

## Which documentation and/or configurations were updated?

loc_el_taher_config.rb, no documentation update necessary

### CODE REVIEW

File: traject_configs/loc_el_taher_config.rb [hunk 1]
```diff
@@ -39,8 +39,8 @@
 
 # Cho Required
 to_field 'id', extract_json('.id')
-to_field 'cho_title', extract_json('.item.title'), strip, arabic_or_none
-to_field 'cho_title', extract_json('.item.other_title[0]'), strip, arabic_or_none
+to_field 'cho_title', extract_json('.item.title'), strip, gsub('/', ''), arabic_or_none
```
Comment (jcoyne) [inline traject_configs/loc_el_taher_config.rb:42]: Should this be in a separate PR?  The PR says this is about edm_type, but this is cho_title?

**Review by jcoyne — CHANGES_REQUESTED** [review]

File: traject_configs/loc_el_taher_config.rb (hunk 1 above)
Comment (jacobthill) [reply traject_configs/loc_el_taher_config.rb:42]: I think its fine in one PR but I have changed the description to reflect the correct scope.

**Review by jcoyne — APPROVED** [review]
